cmd/livepeer_cli/wizard_ticketbroker.go

```diff
@@ -46,20 +47,41 @@ func (w *wizard) deposit() {
 		return
 	}
 
-	fmt.Printf("Current Deposit: %v\n", sender.Deposit)
-	fmt.Printf("Current Reserve: %v\n", sender.Reserve.FundsRemaining)
+	fmt.Printf("Current Deposit: %v\n", eth.FormatUnits(sender.Deposit, "ETH"))
+	fmt.Printf("Current Reserve: %v\n", eth.FormatUnits(sender.Reserve.FundsRemaining, "ETH"))
 
 	fmt.Printf("Enter deposit amount in ETH - ")
-
-	depositAmount := w.readPositiveFloat()
+	fmt.Printf("> ")
+	depositAmount, err := w.in.ReadString('\n')
```

Comment:
Prior to this PR, the `readPositiveFloat()` would keep prompting the user for input if the user provided an invalid input (i.e. negative number or non-number). What do you think about maintaining that behavior but instead of reading a float we would be reading a `big.Int` that represents a "base amount". The helpers would look something like:

```
func (w *wizard) readBaseAmountAndValidate(validate func(in *big.Int) error) *big.Int {
	for {
		fmt.Printf("> ")
		text, err := w.in.ReadString('\n')
		if err != nil {
			log.Crit("Failed to read user input", "err", err)
		}
		if text = strings.TrimSpace(text); text == "" {
			continue
		}
		val, err := eth.ToBaseAmount(text)
		if err != nil {
			log.Error("Error parsing user input", "err", err)
			continue
		}
		if err := validate(val); err != nil {
			log.Error("Invalid user input", "err", err)
			continue
		}
		return val
	}
}

func (w *wizard) readPositiveBaseAmount() *big.Int {
	return w.readBaseAmountAndValidate(func(in *big.Int) error {
		if in.Cmp(big.NewInt(0)) < 0 {
			return errors.New("base amount must be positive")
		}

		return nil
	})
}
```

and the code here would be simplified to

```
// Prompt for deposit
// ...
depositAmount := w.readPositiveBaseAmount()

// Prompt for reserve
// ...
reserveAmount := w.readPositiveBaseAmount()
```